cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java

```diff
@@ -197,15 +197,15 @@ public void execute() {
             createModule(projectPath, moduleName, template);
         } catch (ModuleCreateException e) {
             CommandUtil.printError(errStream,
-                    "Error occurred while creating module : " + e.getMessage(),
+                    "error occurred while creating the module: " + e.getMessage(),
                     null,
                     false);
             return;
         }
 
-        errStream.println("Added new ballerina module at '" + userDir.relativize(projectPath
+        errStream.println("added a new Ballerina module at '" + userDir.relativize(projectPath
                 .resolve(ProjectDirConstants.SOURCE_DIR_NAME)
-                .resolve(moduleName)) + "'");
+                .resolve(moduleName)) + "'.");
```

Comment:
Do we need a full stop at the end even when it is a single sentence starting with a lower case letter?

I find it weird.

The output with and without looks like 
```cmd 
$ ballerina add abc
added a new Ballerina module at 'src/abc'
```
vs
```cmd 
$ ballerina add abc
added a new Ballerina module at 'src/abc'.
```

Checked `git` briefly. AFAICT only the sentences starting with an uppercase letter end with a full stop.

Reply:
Removed the full stop.